### DISCUSSION TITLE
Ignoring a folder after it's been indexed

### DISCUSSION THREAD

**Discussion by jvalskis** [photoprism/photoprism#3996] [Q&A] [answered]
Hello, I added a folder to ignore after it has already been indexed, and would like to remove it from the index. I've seen `photoprism reset` suggested in the FAQ, but I assume it would also destroy all my albums, facial recognition matches and etc. (the command description is not very descriptive about the actual side effects).

I would like to only process that one folder. How can I go about it? I've also read a few old posts about the same topic, and I'm wondering is there a better way than to manually amend the database. If there's still no better way, then could I get a quick rundown on what sort of data needs to be deleted, and what to do afterwards.

**Comment by graciousgrey** [comment, marked as answer]
The reset command applies to your whole library, so it is not suitable to remove a single folder from the index.

The following might work:
- Move the folder you want to ignore outsides of the originals folder.
- Then let PhotoPrism index this folder with the cleanup option enabled. PhotoPrism should then detect that the files are missing and remove them from the index and remove related thumbnails.
- Then move your files back to the originals folder. In subsequent index runs they should not be added to the index in case the folder is listed in the ignore file.

**Reply by jvalskis** [reply]
That worked, thanks.
